### PR TITLE
Refactor TrackedTxsIndexer using ManagedMultiIndexer

### PR DESCRIPTION
**Pull request by maciekzygmunt**
Resolves L2B-4905

### CODE REVIEW

File: packages/database/src/l2-cost/repository.ts [hunk 1]
```diff
@@ -28,24 +27,48 @@ export class L2CostRepository {
   async getWithProjectIdByTimeRange(timeRange: [UnixTime, UnixTime]) {
```
Comment (maciekop-l2b) [inline packages/database/src/l2-cost/repository.ts:27]: Let's apply the same changes as suggested for knex repo

File: packages/database/src/liveness/repository.ts [hunk 2]
```diff
@@ -26,20 +27,42 @@ export class LivenessRepository {
   }
 
   async getWithSubtypeDistinctTimestamp(projectId: ProjectId) {
```
Comment (maciekop-l2b) [inline packages/database/src/liveness/repository.ts:29]: Let's apply the same changes as suggested for knex repo

**Review by maciekop-l2b — CHANGES_REQUESTED** [review]
Great work!

Some minor things to correct.

File: packages/backend/src/modules/tracked-txs/TrackedTxsIndexer.ts [hunk 3]
```diff
@@ -1,250 +1,121 @@
...
-    private readonly configRepository: TrackedTxsConfigsRepository,
-    private readonly configs: TrackedTxConfigEntry[],
-    private readonly minTimestamp: UnixTime,
-  ) {
-    super(logger, [parentIndexer])
-    this.enabledUpdaters = pickBy(updaters, notUndefined)
+export class TrackedTxsIndexer extends ManagedMultiIndexer<TrackedTxConfigEntry> {
+  constructor(private readonly $: Dependencies) {
+    const name = 'tracked_txs_indexer'
+    const logger = $.logger.tag(name)
```
Comment (antooni) [inline packages/backend/src/modules/tracked-txs/TrackedTxsIndexer.ts:32]: is this additional tagging needed?

**Review by maciekop-l2b — APPROVED** [review]

**Review by antooni — APPROVED** [review]

**Pull request merged**